### DISCUSSION TITLE
Rename judgements in mania to be less confusing when switching between VSRGs, also add a new rank as a sort of legacy SS

### DISCUSSION THREAD

**Discussion by mooshiros** [ppy/osu#29246] [Ideas] [answered]
In other rhythm games (notably Etterna and Quaver because they're the two of the three biggest PC VSRGs along with o!m), the judgements go

Marvelous -> Perfect -> Great -> Good -> Bad/Okay (Etterna/Quaver) -> Miss

while in osu mania it goes

Perfect -> Great -> Good -> Ok -> Meh -> Miss

Since this is almost the same set of judgements as the other rhythm games, the fact that the judgements don't correspond to each other is quite confusing. I suggest renaming the judgements (shifting them down, getting rid of "Meh", and adding "Marvelous") in order to fit better with other VSRGs. I realize that this will then create a difference between o!m and the other osu game modes, and honestly I feel like the judgements should be renamed in all gamemodes as it is still confusing in those (not as much though because the communities are more self contained than the o!m community). I also don't think it's particularly bad if the judgement names don't match up between mania and the other gamemodes as it has more judgements anyways. Obviously this issue doesn't exist in stable as all judgements are just numbers. 

As for the "legacy SS" I mentioned in the title, now that lazer requires an MFC to get an SS rather than the PFC that was required in stable, I think an extra rank should be added for PFCs set on lazer, as well as a display of your combined MFC+PFC count on your profile to emulate what your SS count would have been on stable.

**Comment by peppy** [comment, marked as answer]
The best fix for this is to make a skin which changes the naming.

Having something above "perfect" only makes sense to people who are deep in the lore, and not to new players.

This has been discussed to death and only makes sense as a proposal if you've only played the two games you mention (and/or stepmania). No others use such naming and we don't intend to match it officially. But of course you can skin it!

**Reply by SupDos** [reply]
> make a skin which changes the naming

Slightly related - would there ever be a possibility of allowing skins to change the actual name, and colour, of the judgements (as they show on the score results card and judgement counter) or would that be something that would be kept consistent for design purposes?

The above mania example would be one case, but I'm also sure some standard players might prefer the 300, 100, 50, etc names instead of the current ones.

scuffed image edits as examples

![image](https://github.com/user-attachments/assets/3cfe8483-8802-4124-93d5-05105f354b06)

![image](https://github.com/user-attachments/assets/4ff4c291-a27e-452b-8382-7b8c5d02f924)

![image](https://github.com/user-attachments/assets/88730a4b-a96e-4b3e-896d-e80437643c0a)

**Reply by mooshiros** [reply]
Do you have any thoughts on the legacy SS counter I mentioned?
